app/components/Icon.js

```diff
@@ -5,9 +5,10 @@ import { colors } from '../styles/colors';
 
 /**
  * Create an icon component.
- * @param {string} name - The icon name.
- * @param {number} [size=30] - The icon size.
- * @param {string} [color=colors.primary] - The color to use for the icon.
+ * @param {Object} props
```

Comment:
For the props argument on these components wrappers, I would make add a description detailing which component this is wrapping and that those props will be applied. 
Example:
props - All properties will be passed to the MaterialCommunityIcon component